src/lib/utils/index.ts

```diff
@@ -261,3 +262,32 @@ export const assertGetChainAdapter = (
 
   return adapter
 }
+
+export const ALLOWED_PRICE_IMPACT_LOW: string = '1' // 1%
+export const ALLOWED_PRICE_IMPACT_MEDIUM: string = '3' // 3%
+export const ALLOWED_PRICE_IMPACT_HIGH: string = '5' // 5%
+export const BLOCKED_PRICE_IMPACT_NON_EXPERT: string = '15' // 15%
```

Comment:
Still in the process of grasping this PR but assuming this is allowed only after accepting an expert mode CTA, I would avoid the double negative (blocked / non-expert)
```suggestion
export const ALLOWED_PRICE_IMPACT_EXPERT: string = '15' // 15%
```